fighters/link/src/status.rs

```diff
@@ -258,6 +258,19 @@ pub unsafe fn pre_specialhi_end(fighter: &mut L2CFighterCommon) -> L2CValue {
 pub unsafe fn specialhi_end(fighter: &mut L2CFighterCommon) -> L2CValue {
     WorkModule::enable_transition_term(fighter.module_accessor, *FIGHTER_STATUS_TRANSITION_TERM_ID_FALL_SPECIAL);
     WorkModule::enable_transition_term(fighter.module_accessor, *FIGHTER_STATUS_TRANSITION_TERM_ID_WAIT);
+    if fighter.global_table[SITUATION_KIND] == SITUATION_KIND_GROUND {
```

Comment:
this can just be `fighter.is_situation(*SITUATION_KIND_GROUND)` iirc

Reply:
it can